src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredIsMan = people.filter(person => person.sex === 'm');
+
+  const filteredByCentury = (century !== undefined)
+    ? filteredIsMan.filter(person => Math.ceil(person.died / 100) === century)
+    : filteredIsMan;
```

Comment:
In case when you'll have this century, your code will filter array twice, but there is a way how to do it only once

Just use one filter method, and use ternary operator inside it to get needed condition